spec/inputs/jdbc_spec.rb

```diff
@@ -506,4 +506,39 @@
     end
   end
 
+  describe "config option force_lowercase_keys behaviour" do
+    let(:settings) { { "statement" => "SELECT * FROM ttt" } }
+    let(:events) { [] }
+
+    before do
+      db.create_table(:ttt) do
+        Integer(:num)
+        String(:somestring)
+      end
+      db[:ttt].insert(:num => 42, :somestring => "This is a string")
+      plugin.register
+    end
+
+    after do
+      plugin.stop
+      db.drop_table(:ttt)
+    end
+
+    context "when force_lowercase_keys is off (default)" do
+      it "the field names are lower case" do
+        plugin.run(events)
+        values = events.first.to_hash.values_at("num", "somestring")
```

Comment:
my suggestion is to make the spec target the names of the keys, so something similar to:

`expect(events.first.to_hash.keys).to eq(["num", "somestring"])`